core/modules/parser/QSMySqlListener.cc

```diff
@@ -2112,6 +2401,44 @@ class BetweenPredicateAdapter :
 };
 
 
+class IsNullPredicateAdapter :
+        public AdapterT<IsNullPredicateCBH, QSMySqlParser::IsNullPredicateContext>,
```

Comment:
Private inheritance for `QSMySqlParser::IsNullPredicateContext>`? Should the see made more visible by putting it on a separate line?

Reply:
Please, disregard my comment above!